components/userPages/Login/LogInForm.js

```diff
@@ -23,20 +25,24 @@ export default function UserLoginForm() {
     if (!Obj.err) {
       setUserObj({ ...Obj, logedIn: true });
     } else {
-      alert(Obj.message)
+      alert(Obj.message);
     }
   }
 
   return (
-    <form onSubmit={handleSubmit}>
-      <Input _label={"E-mail: "} _type="email" _setState={setUserEmail} />
-      <Input
-        _minLgth={4}
-        _label={"password"}
-        _type="password"
-        _setState={setPassword}
-      />
-      <button type="submit">login</button>
-    </form>
+    <div className={styles["login"]}>
+      <form onSubmit={handleSubmit}>
+        <div>
+          <Input _label={"E-mail"} _type="email" _setState={setUserEmail} />
+          <Input
```

Comment:
The `Input` component assumes that there is an `_id` property, but none is passed here.

This is one of those situations where typescript would have helped.